[docs] add account member_limit to working together doc (#13413)
expo/expo: docs/pages/accounts/working-together.md

## Adding Members

> You can invite new members to your Personal Account, or any account you administrate, from the [Members page](https://expo.io/settings/members) in your dashboard. You can only add users with Expo accounts as members; you can direct them to [https://expo.io/signup](https://expo.io/signup) if they don't have an account yet. By default, you have up to 100 members and pending invitations for an account.

Reviewer: ```suggestion
You can invite new members to your Personal Account, or any account you administrate, from the [Members page](https://expo.io/settings/members) in your dashboard. You can only add users with Expo accounts as members; you can direct them to [https://expo.io/signup](https://expo.io/signup) if they don't have an account yet. You may have up to 100 members and pending invitations combined for a single account.
```

"By default" makes it seem like there's a way to edit this limit. Just slightly reworded this. I'd love it if you'd make this change or similar. Then feel free to merge.

Author: Got it, language sounds much better!
